sdl_fract_viewer_2/Makefile

```
@@ -0,0 +1,28 @@
+CC=gcc
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)